### ISSUE TITLE
Inconsistency in vtk files from cli and GUI

### ISSUE THREAD

**Issue by tahsbahs** [firelab/windninja#502]
I noticed an inconsistency between running windninja from cli and from the GUI. 

I have stripped down the configuration file to this:

```
num_threads                = 1
elevation_file             = west_ireland_utm.tif
output_path                = .
initialization_method      = domainAverageInitialization
input_speed                = 10
input_speed_units          = mps
output_speed_units         = mps
input_direction            = 270
input_wind_height          = 200.0
units_input_wind_height    = m
output_wind_height         = 200.0
units_output_wind_height   = m
vegetation                 = brush
mesh_resolution            = 500
units_mesh_resolution      = m
write_vtk_output           = true
```

for vegetation I use either "grass" or "brush". When loading the data and averaging over the domain I get the following profiles (y axis wind speed, x axis height):

<img width="298" alt="image" src="https://github.com/firelab/windninja/assets/43064125/0d4267eb-afd4-4627-b21c-81b4c36e6e48">

The profiles are identical. I have confirmed that the vtk files that are out up are also indeed identical. 

Running the same (to my best knowledge) configuration using the GUI produces results below:

<img width="305" alt="image" src="https://github.com/firelab/windninja/assets/43064125/e40de655-e746-4fb3-9168-6a27716f1f19">

Here I can see a difference in the wind profile which I expected. 

We have compiled a version of wind ninja on linux and did not see the problem there. 

Any idea what could cause that issue?

I ran this on wind ninja 3.8.1 not 3.9.0. Need to look into a vtk reader that can do binary as well.

**Comment by nwagenbrenner** [comment]
We fixed a bug in 3.9.0 that was causing the vegetation to be set incorrectly in the GUI for some runs. 

https://github.com/firelab/windninja/blob/master/NEWS.md
https://github.com/firelab/windninja/issues/486
https://github.com/firelab/windninja/commit/2533d6bac4f57d00080e8a4404473578a4ad3cae

I'm closing the issue, but if you continue to see this bug in either 3.9.0 or the master branch, please reopen and let us know.

**Comment by tahsbahs** [comment]
Thank you. I can confirm that it has been fixed in 3.9.0
